canonical-web-and-design/ubuntu.com
# Account users/add new user button

## Done

- Add "Add new user" button along with the barebones modal implementation 
  - validation and further functionality will be implemented separately

## QA

- Login
- Go to /advantage/users
- Click "Add new user"
- Verify that the modal has been displayed containing correct fields


## Issue / Card

https://github.com/canonical-web-and-design/commercial-squad/issues/196

## Screenshots

[If relevant, please include a screenshot.]
![image](https://user-images.githubusercontent.com/7452681/132530221-f6f68985-0f20-4cbe-a9b8-abc83c9b9ed5.png)
![image](https://user-images.githubusercontent.com/7452681/132530242-e99fb9db-0cdf-4852-9c8a-b78ba3e64262.png)

## Zeplin spec
Below you will see design of a separate page, but It's been agreed upon to implement as a modal instead:
https://zpl.io/adywyop
